docs/content/integrations/airflow/reference.mdx

```diff
@@ -0,0 +1,256 @@
+    "email": ["airflow@example.com"],
+    "email_on_failure": False,
+    "email_on_retry": False,
+}
+
+dag, tasks = make_airflow_dag(
+    module_name="docs_snippets.integrations.airflow.hello_cereal",
+    job_name="hello_cereal_job",
+    dag_kwargs={"default_args": DEFAULT_ARGS, "max_active_runs": 1},
+)
+```
+
+If you run this code interactively, you'll see that `dag` and `tasks` are ordinary Airflow objects, just as you'd expect to see when defining an Airflow pipeline manually:
+
+```python
+>>> dag
+<DAG: hello_cereal_job>
+>>> tasks
+[<Task(DagsterPythonOperator): hello_cereal>]
+```
+
+Like other Airflow DAG definition files, this should go inside `$AIRLFLOW_HOME/dags`. The `docs_snippets.integrations.airflow.hello_cereal` module that's passed as the value for the `module_name` argument must be importable via the sys.path.
+
+After this, the DAG should show up inside Airflow:
+
+<Image
+alt="intro_airflow_one.png"
+src="/images/deploying/intro_airflow_one.png"
+width={1272}
+height={868}
+/>
+
+### Running Containerized
+
+The approach above runs each op inside an operator that's similar to the Airflow PythonOperator. If you instead want to containerize your Dagster job and run it using an operator that's similar to the Airflow DockerOperator, you can use <PyObject module="dagster_airflow" object="make_airflow_dag_containerized" />.
+
+As in the uncontainerized case, you'll put a new Python file defining your DAG in the directory in which Airflow looks for DAGs:
+
+```python file=/integrations/airflow/containerized.py
+# pylint: disable=unused-variable
+
+import datetime
+
+from dagster_airflow.factory import make_airflow_dag_containerized
+
+DEFAULT_ARGS = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime.datetime(2019, 11, 7),
+    "email": ["airflow@example.com"],
+    "email_on_failure": False,
+    "email_on_retry": False,
+}
+
+dag, steps = make_airflow_dag_containerized(
+    module_name="docs_snippets.integrations.airflow.hello_cereal",
+    job_name="hello_cereal_job",
+    image="dagster-airflow-demo-repository",
+    dag_kwargs={"default_args": DEFAULT_ARGS, "max_active_runs": 1},
+)
+```
+
+The `image` argument is the name of the Docker image. Running in a containerized context requires a persistent intermediate storage layer available to the Dagster containers, such as a network filesystem, S3, or GCS. You can pass `op_kwargs` through to the the **`DagsterDockerOperator`** to use custom TLS settings, the private registry of your choice, etc., just as you would configure the ordinary Airflow **`DockerOperator`**.
+
+If you want your containerized job to be available to Airflow operators running on other machines (for example, in environments where Airflow workers are running remotely) you'll need to push your Docker image to a Docker registry so that remote instances of Docker can pull the image by name, or otherwise ensure that the image is available on remote nodes.
```

Comment:
```suggestion
If you want your containerized job to be available to Airflow operators running on other machines - for example, in environments where Airflow workers are running remotely - you'll need to push your Docker image to a Docker registry so that remote instances of Docker can pull the image by name, or otherwise ensure that the image is available on remote nodes.
```